nixos/modules/services/misc/lidarr.nix

```diff
@@ -44,6 +50,10 @@ in
   };
 
   config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' 0700 ${cfg.user} ${cfg.group} - -"
```

Comment:
Note that if the default value of `dataDir` is used then `/var/lib/lidarr/.config` and `/var/lib/lidarr` will be owned by `root`. Is this a problem?

Reply:
Works fine on my server where I run with the patch, it's exactly the same behavior as Sonarr and Radarr :smiley:

Reply:
And yes, you're right, those directories are owned by root. But Lidarr doesn't complain about that.

Reply:
Sounds good, just wanted to make sure. Thanks!